features/device-config/monitor/src/main/java/org/opennms/features/deviceconfig/monitors/DeviceConfigMonitor.java

```diff
@@ -150,6 +152,9 @@ public PollStatus poll(MonitoredService svc, Map<String, Object> parameters) {
             status.setDeviceConfig(new DeviceConfig());
             return status;
         }
+        parameters.put(EventConstants.PARM_DEVICE_CONFIG_BACKUP_START_TIME, System.currentTimeMillis());
```

Comment:
monitor may not have reference for `events.api` in osgi world, likely that smoke tests will fail.
Better to use `DeviceConfigConstants.java` for this